resources/views/bread/edit-add.blade.php

```diff
@@ -90,16 +90,22 @@
                                         @endif
                                         <input type="file" name="{{ $row->field }}">
                                     @elseif($row->type == "select_dropdown")
```

Comment:
@marktopper, please note that these lines were moved to their own view partials in v0.11.  This will likely have to be updated manually before the release.